src/styles/style.css

```diff
@@ -0,0 +1,13 @@
+@import './blocks/stars.css';
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using the `*` selector as it impacts performance. It's better to set styles only for elements that require them. Instead of zeroing out margins and paddings for all elements, consider applying them to specific elements or use a more targeted reset strategy.